src/core/datatypes/trees/TopologyNode.h

```diff
@@ -219,4 +219,6 @@ namespace RevBayesCore {
     };
 }
 
+std::pair<double,double> getStartEndAge(const RevBayesCore::TopologyNode& node);
```

Comment:
Why is this a global method and not a member method?

Reply:
As we talked about on the call, global methods are fine.